catalogue/webapp/components/IIIFImage/IIIFImage.tsx

```diff
@@ -33,43 +33,59 @@ const IIIFImage: FC<Props> = ({
   onLoadingComplete,
   priority = false,
   layout,
-  width = 300,
 }) => {
   const sizesString = sizes
     ? convertBreakpointSizesToSizes(sizes).join(', ')
     : undefined;
 
+  // The Nextjs Image component has an experimental 'raw' layout feature
+  // which will be rendered as a single image element with no wrappers, sizers or other responsive behavior.
+  // We may be able to use this in future but, until then, render our own img element.
+  if (layout === 'raw') {
+    return (
+      <img
+        src={iiifImageTemplate(image.contentUrl)({
+          size: `${image.width},`,
+        })}
+        srcSet={''}
+        sizes={sizesString}
+      />
+    );
+  }
+
+  if (layout === 'fixed') {
+    return (
+      <Image
+        layout={layout}
+        sizes={sizesString}
+        src={image.contentUrl}
+        alt={image.alt || ''}
+        loader={IIIFLoader}
+        onLoadingComplete={onLoadingComplete}
+        width={image.width}
+        height={image.height}
+        priority={priority}
+        placeholder="blur"
+        blurDataURL="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNctWxZEQAGYgJqdIxNSwAAAABJRU5ErkJggg=="
```

Comment:
For those of us who don't speak base64-encoded PNG, what does this do?

We have a file where hardcoded images live (which you may not have discovered yet); perhaps it should live there? https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/utils/backgrounds.ts

Reply:
Oooh yes, I'll move it! It's what shows whilst the image is loading, it contains the colour I chose for it (it's just a grey/transparent square at the moment).